_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -22,28 +22,19 @@ acknowledgments:
 
 ## Applicability
 
-Any [WAI-ARIA 1.1 state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is not empty (""), and that is specified on an HTML or SVG element.
+The rule applies to any [WAI-ARIA 1.1 state or property][] that is not empty (`""`), and is specified on an HTML or SVG element.
 
 ## Expectation
 
-Each test target has a valid value according to its [WAI-ARIA 1.1 value type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value).
-
-For value types `ID Reference` and `ID Reference List` for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState) at least one of the elements with the given ids exists in the same [document tree](https://dom.spec.whatwg.org/#document-trees) or [shadow tree](https://dom.spec.whatwg.org/#shadow-trees) as the element that specifies the target attribute.
-
-For value type `URI` the value matches the [generic URI syntax](https://www.ietf.org/rfc/rfc3986.txt).
-
-**Note:** Only for [WAI-ARIA required properties](https://www.w3.org/TR/wai-aria-1.1/#requiredState) with value types `ID Reference` and `ID Reference List` is there a requirement that the elements with the given ids actually exists. For non-required properties, this is not a requirement.
-
-**Note:**
-For value type `URI`, this rule does not require that the destination URI exists.
+Each test target has a valid value according to its [WAI-ARIA 1.1 value type][]. For [WAI-ARIA required properties][] with [value type][] of `ID reference` and `ID reference list` at least one of the referenced elements with the given ids should exist in the same [document tree][] or [shadow tree][].
```

Comment:
Can you please review the comments again? This rule stats that these only need to exist if the ARIA attribute is required for that role.